cmd/manifests/README.md

```diff
@@ -0,0 +1,88 @@
+# Manifests
+
+The `manifests` command generates the YAML manifests required to use CAPV. It has several benefits over the previous method:
+
+* Links to Kustomize as a library, obviating the requirement for the `kustomize` binary
+* Supports [_virtual patches_](#Virtual-patches) for Kustomize
+* The `cluster`, `machines`, or `machineset` manifests are generated from the version of the API types with which the program is built, removing the risk of drift between templates and API versions
+* Avoids general issues associated with shell scripts
+
+## Getting started
+
+Running the `manifests` command requires access to both the CAPI provider's and CAPI framework's config directories. While more options are available, these are the only requirements. For example:
+
+```shell
+./manifests -config-dir "$(pwd)/config/default" \
+            -config-dir "$(pwd)/vendor/sigs.k8s.io/cluster-api/config/default"
+```
+
+```shell
+$ ls -al
+-rw-r--r--  1 akutz  staff    19K Jul 23 12:00 addons.yaml
+-rw-r--r--  1 akutz  staff   484B Jul 23 12:00 cluster.yaml
+-rw-r--r--  1 akutz  staff   640B Jul 23 12:00 machines.yaml
+-rw-r--r--  1 akutz  staff   936B Jul 23 12:00 machineset.yam
+-rw-r--r--  1 akutz  staff    99K Jul 23 12:00 provider-components.yaml
+```
+
+For a full list of the flags and options, use `./manifests -?`.
+
+## Virtual patches
+
+The `manifests` program introduces the concept of _virtual patches_ for Kustomize.
+
+_A virtual patch is any file located in a kustomization root that ends with `.template`._
```

Comment:
What is "kustomization root" exactly? A directory with a `kustmization.yaml` file?